src/lib.rs

```diff
@@ -7,13 +7,19 @@
 //! version numbers need updating:
 //!
 //! * TOML examples in the `README.md` files that show how to add a
-//!   dependency on your crate. See [`assert_markdown_deps_updated`].
+//!   dependency on your crate, gated behind the
```

Comment:
We should definitely mention the feature flags somewhere, but I think it feels a bit unnatural here. That is, I read the sentence as

> TOML examples [...] gated behind the "markdown_deps_updated" feature

which makes me think that the TOML examples themselves are gated behind the new feature. They're clearly not, but i know what you mean :-)

Could you please take out the gated part again and then add it to a new section called "Cargo Features"?

Reply:
I moved this to a separate section, so this should be good now.